### ISSUE TITLE
Inser duplicado

### ISSUE THREAD

**Issue by krefects** [oscarotero/simple-crud#34]
Estou inserindo dados na tabela. mais o registro inserido fica duas vezes
$id = $db->noticias_categorias
    ->insert()
    ->data([
        'titulo' => 'titulo_teste ',
    ]);

	
----- ROW 1--
33
titulo_teste
---- ROW 2 duplicado --

32
titulo_teste

**Comment by oscarotero** [comment]
Ten que haber un erro nalgun outro sitio do teu código. Con esto que me envías non debería inserir o rexistro nin unha soa vez (falta executar `->run()`)

**Comment by krefects** [comment]
Sim, estou executando o run() e insere normalmente no banco de dados , porém insere dois registros, invés de um.

**Comment by oscarotero** [comment]
É posible que esteas executando `run()` dúas veces? Podo ver o código completo?

**Comment by krefects** [comment]
include "conf.php"; // Vendor
$databaseConexao = \SISTEMA\database\conexao::getConexao();

use SimpleCrud\SimpleCrud;

$db = new SimpleCrud($databaseConexao);

$newPost = $db->noticias_categorias->create(['titulo' => 'Titulo de teste 001']);
$newPost->save();


------------ RESULTADO --------
  | Edita | Copiar | Apagar | 51 | Titulo de teste 001 | NULL
-- | -- | -- | -- | -- | -- | --
  | Edita | Copiar | Apagar | 50 | Titulo de teste 001 | NULL

Veia que o mesmo item foi inserido duas vezes

**Comment by oscarotero** [comment]
Acabo de facer unha proba e son incapaz de reproducir este erro. Sigo pensando que non é un erro de SimpleCrud.
